lib/cylc/batch_sys_handlers/slurm.py

```diff
@@ -74,5 +74,9 @@ def get_fail_signals(self, job_conf):
         """
         return ["EXIT", "ERR", "XCPU"]
 
```

Comment:
Can you make the same POLL_CMD and POLL_CMD_TMPL style changes as per the other batch sys handlers in slurm.py too.